### ISSUE TITLE
Misleading Filter Button Semantics with aria-haspopup

### ISSUE THREAD

**Issue by tobiasberge** [shopware/platform#14790]
### Shopware Version

v6.7.7.0

### Affected area / extension

Platform(Default)

### Actual behaviour

Filter disclosure buttons such as “Manufacture” “Material”, “Shipping Restrictions” have `aria-haspopup="true"` applied, suggesting through the accessibility tree that a “menu” with “menuitems” is present, and will follow an expected keyboard pattern, When the buttons are triggered content is revealed that does not have “menu” or “menuitem” roles applied. 

### Expected behaviour

Remove “haspopup” from the buttons, applying only aria-expanded attributes so that the role of the button, and revealed contents, are correctly communicated through the accessibility tree, and screen-reader users are not instructed to use a keyboard pattern that is not present/functional (when “haspopup” is present screen-reader users are instructed to ‘press space to open the menu, then use up/down arrow keys to navigate’ this instruction is inaccurate given the disclosure, rather than menu, implementation). 

### How to reproduce

* Open the storefront
* Navigate the filter buttons with the screenreader, e.g. VoiceOver
* The buttons suggest a "collapsed menu popup"
